cinema/tests/test_order_api.py

```diff
@@ -73,12 +71,8 @@ def test_movie_session_detail_tickets(self):
             f"/api/cinema/movie_sessions/{self.movie_session.id}/"
         )
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertEqual(
-            response.data["taken_places"][0]["row"], self.ticket.row
-        )
-        self.assertEqual(
-            response.data["taken_places"][0]["seat"], self.ticket.seat
-        )
+        self.assertEqual(response.data["taken_places"][0]["row"], self.ticket.row)
+        self.assertEqual(response.data["taken_places"][0]["seat"], self.ticket.seat)
```

Comment:
These assertions verify that the taken places for the movie session are correctly retrieved, ensuring the API returns the expected data structure.